test/it/user-profile-update.js

```diff
@@ -0,0 +1,43 @@
+const expect = require('chai').expect;
+const utils = require('../utils');
+const okta = require('../../');
+let orgUrl = process.env.OKTA_CLIENT_ORGURL;
+
+if (process.env.OKTA_USE_MOCK) {
+  orgUrl = `${orgUrl}/user-profile-update`;
+}
+
+const client = new okta.Client({
+  orgUrl: orgUrl,
+  token: process.env.OKTA_CLIENT_TOKEN
+});
+
+describe('User API Tests', () => {
+  it('should update the user profile', async () => {
+    // 1. Create a user
+    const newUser = {
+      profile: {
+        firstName: 'John',
+        lastName: 'Profile-Update',
+        email: 'john-profile-update@example.com',
+        login: 'john-profile-update@example.com'
+      },
+      credentials: {
+        password: { value: 'Abcd1234' }
+      }
+    };
+
+    let queryParameters = { activate : 'false' };
+    const createdUser = await client.createUser(newUser, queryParameters);
+    utils.validateUser(createdUser, newUser);
+
+    // 2. Update the user profile
+    createdUser.profile.nickName = 'Batman';
+    const profileUpdateUser = await createdUser.update();
+    expect(profileUpdateUser.lastUpdated).to.be.gt(createdUser.lastUpdated);
+    expect(profileUpdateUser.profile.nickName).to.equal('Batman');
```

Comment:
This check is _less_ valid for the 'update' method if `update()` return itself.
for example in the Java SDK, this is equivalent to:

```
createdUser.profile.nickName = 'Batman'
assert createdUser.profile.nickName == 'Batman'
```